test/integration/narrow/routes/enrich-request.test.js

```diff
@@ -3,7 +3,7 @@ const db = require('../../../../app/data')
 
 const { ADMINISTRATIVE } = require('../../../../app/debt-types')
```

Comment:
Won't let me comment on them because you didn't make changes to those lines directly.

Tests:

- `GET /enrich-request route returns 404 view when invoiceNumber does not exist in the database`
- `GET /enrich-request route returns 404 view when request has been enriched already`

Need their payloads changed to incl. `paymentRequestId`

Also a similar test is needed `when paymentRequestId does not exist in the database`, i.e. a valid `invoiceNumber` but non-matching `paymentRequestId`